microsoft-edge/web-platform/translator-api.md

```diff
@@ -0,0 +1,394 @@
+
+* **Improved privacy:** The data input into the model never leaves the device, and isn't collected to train AI models.
+
+The translation models for specified language pairs are downloaded the first time the API is used in Microsoft Edge, and are subsequently shared across all websites in the browser.  The models are accessed via a straightforward web API that doesn't require knowledge of third-party frameworks, and doesn't require Artificial Intelligence (AI) or Machine Learning (ML) expertise.
+
+
+<!-- ====================================================================== -->
+## Alternatives to the Translator API
+
+You can send network requests to cloud-based translation services with more sophisticated capabilities, including neural translation; see [Azure AI Translator documentation](/azure/ai-services/translator/).
+
+As an on-device alternative, the Prompt API serves more custom, multi-language prompt-engineering scenarios, with a small language model that's built into Microsoft Edge; see [Prompt a built-in language model with the Prompt API](./prompt-api.md).
+
+
+<!-- ====================================================================== -->
+## Disclaimer
+
+Like other machine translation models, the expert models in Microsft Edge can potentially produce translations that are unfair, unreliable, or offensive.  To learn more about the limitations of these machine translation models, see [Limitations](/azure/ai-foundry/responsible-ai/translator/transparency-note#limitations) in _Azure AI Translator Transparency Note_.
+
+
+<!-- ====================================================================== -->
+## Model availability
+
+An initial download of the model will be required the first time a website calls a built-in AI API.  You can monitor the model download by using the monitor option when creating a new Translator API session; see [Monitor the progress of the model download (monitor:)](#monitor-the-progress-of-the-model-download-monitor), below.
+
+
+<!-- ====================================================================== -->
+## Enable the Translator API
+
+To use the Translator API in Microsoft Edge:
+
+1. Make sure you're using the latest version of Microsoft Edge Canary or Dev (version 138.0.3309.2 or newer<!-- todo: add correct version -->).  See [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider).
+
+1. In Microsoft Edge Canary or Dev, open a new tab or window and go to `edge://flags/`.
+
+1. In the search box, at the top of the page, enter **Experimental translation API**.
+
+   The page is filtered to show the matching flag.
+
+1. Under **Experimental translation API**, select **Enabled**:
+
+   ![Flags page of browser](./translator-api-images/flags-translator-api.png)
+
+1. Restart Microsoft Edge Canary or Dev.
+
```

Comment:
Chrome Canary 143 adds flag:
**Translation API streaming split by sentence**
Enables sentence-split streaming for on-device translation API. – Mac, Windows, Linux
`#translation-api-streaming-by-sentence`

The affected article section mentions 1 of the 2 flags:
https://review.learn.microsoft.com/en-us/microsoft-edge/web-platform/translator-api?branch=pr-en-us-3566#enable-the-translator-api
**Experimental translation API**
Enables the on-device language translation API. See https://github.com/WICG/translation-api/blob/main/README.md – Mac, Windows, Linux
`#edge-translation-api`